notebook/supermusr-plotting-frames.ipynb

```diff
@@ -0,0 +1,127 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 40,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# These values are set here and used further down\n",
+    "filename = \"trace-Tue 26 Jul 10-57-38 BST 2022.h5\"\n",
```

Comment:
I ran out of time trying to get this to work today, but is is possible to use a path to the network data here? (I assume the issue is the Windows style $ suffix)

Reply:
I'm not sure, I am guessing it "should" work.

Reply:
It does work I tested it on IDAaaS. Specifically with data in the /archive/Backups$/ folder.